src/item.h

```diff
@@ -564,6 +564,16 @@ class item : public visitable<item>
          * @param allow_bucket Allow filling non-sealable containers
          */
         bool is_container_full( bool allow_bucket = false ) const;
+        /**
+         * Mix the rot from one item to another, sets to max relative rot
+         * @param other Item to take temperature from
+         */
+        void mix_rot( const item &other );
```

Comment:
Do we need those `mix_` function to be public and available to all? They are used only at one place. Are you expecting to use them elsewhere? Maybe their code can be used directly instead of having it in separate (and public) functions.

Reply:
In my mind at the time it seemed like they might be useful for later.  A follow-up PR is going to fix crafting temperatures such that crafting results inherit the average temperature of their components.  I've thought about it more since and these functions probably aren't necessary there either.  Will fix.